### DISCUSSION TITLE
I have trouble generating images with the openOutpaint extension (AUTOMATIC1111 Webui on Colab)

### DISCUSSION THREAD

**Discussion by Corpostar** [TheLastBen/fast-stable-diffusion#1988] [Q&A]
Hello everyone, I hope you are having a good day. I am using the openOutpaint extension and the sd-v-1-5-inpainting.ckpt model on AUTOMATIC1111 Webui on Colab, but I cannot generate any image (using Edge, I also tried with Firefox). I have almost no knowledge of programming. The problem is that the openOutpaint extension doesn’t work on the webui. Previously, I had used other adaptations of this colab and openOutpaint worked fine.
![Capturanueva](https://user-images.githubusercontent.com/123031254/231928401-a34bd236-b137-47b5-823c-3d9dc15da716.JPG)
I have tried deleting and reinstalling sd from drive, and also modifying the webui-user.sh file so that: 
```# Commandline arguments for webui.py, for example: export COMMANDLINE_ARGS=“–medvram --opt-split-attention” export COMMANDLINE_ARGS=“–allow-code --xformers --skip-torch-cuda-test --no-half-vae --api --ckpt-dir /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/stable-diffusion-checkpoints”```  And also the webui-user.bat file with the same thing just in case, although I think colab uses linux. I have also tried reinstalling sd from drive without modifying those files and it still doesn’t work. I always get this error: 
/Running on local URL: https://dcb8-34-73-233-140.ngrok-free.app

✔ Connected
Startup time: 3.3s (load scripts: 0.2s, create ui: 2.3s, scripts app_started_callback: 0.5s).
API error: POST: https://dcb8-34-73-233-140.ngrok-free.app/sdapi/v1/img2img {'error': 'RuntimeError', 'detail': '', 'body': '', 'errors': "There is no current event loop in thread 'AnyIO worker thread'."}
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /usr/local/lib/python3.9/dist-packages/anyio/streams/memory.py:94 in receive │
│                                                                              │
│ /usr/local/lib/python3.9/dist-packages/anyio/streams/memory.py:89 in         │
│ receive_nowait                                                               │
╰──────────────────────────────────────────────────────────────────────────────╯
WouldBlock

During handling of the above exception, another exception occurred:

╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /usr/local/lib/python3.9/dist-packages/starlette/middleware/base.py:43 in    │
│ call_next                                                                    │
│                                                                              │
│ /usr/local/lib/python3.9/dist-packages/anyio/streams/memory.py:114 in        │
│ receive                                                                      │
╰──────────────────────────────────────────────────────────────────────────────╯
EndOfStream

During handling of the above exception, another exception occurred:

╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/api/api.py:145 in  │
│ exception_handling                                                           │
│                                                                              │
│   144 │   │   try:                                                           │
│ ❱ 145 │   │   │   return await call_next(request)                            │
│   146 │   │   except Exception as e:                                         │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │        call_next = <function                                             │ │
│ │                    BaseHTTPMiddleware.__call__.<locals>.call_next at     │ │
│ │                    0x7f75a810c280>                                       │ │
│ │                e = RuntimeError("There is no current event loop in       │ │
│ │                    thread 'AnyIO worker thread'.")                       │ │
│ │ handle_exception = <function api_middleware.<locals>.handle_exception at │ │
│ │                    0x7f74c73f4040>                                       │ │
│ │          request = <starlette.requests.Request object at 0x7f759a52fa00> │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /usr/local/lib/python3.9/dist-packages/starlette/middleware/base.py:46 in    │
│ call_next                                                                    │
│                                                                              │
│                           ... 25 frames hidden ...                           │
│                                                                              │
│ /usr/lib/usr/lib/python3.9/asyncio/locks.py:81 in __init__

80 │   if loop is None:
❱  81 │   │   self._loop = events.get_event_loop()
82 │   else:

╭──────────────────────────── locals ─────────────────────────────╮
│ loop = None                                                     │
│ self = <asyncio.locks.Lock object at 0x7f759a083310 [unlocked]> │
╰─────────────────────────────────────────────────────────────────╯

/usr/lib/python3.9/asyncio/events.py:642 in get_event_loop

641 │   if self._local._loop is None:
❱ 642 │   │   raise RuntimeError('There is no current event loop in thread '
│ 643 │   │   │   │   │   │      % threading.current_thread().name)

╭───────────────────────────────── locals ─────────────────────────────────╮
│ self = <asyncio.unix_events._UnixDefaultEventLoopPolicy object at        │
│        0x7f759b468e50>                                                   │
╰──────────────────────────────────────────────────────────────────────────╯

RuntimeError: There is no current event loop in thread 'AnyIO worker thread'.

. Does anyone know how to fix this problem? Note: I found another similar topic in this forum, but it had no solution to the problem.

**Comment by meunumerotim333** [comment]
Maybe its too complex to Colab. I wish this could work properly too.
